content/05kapitel.tex

```diff
@@ -320,6 +320,108 @@ \subsection{Figma Designs}
     \label{fig:Feed Overview Page Light Mobile}
 \end{figure}
 
+\subsection{Aktivitätsdiagramme}
+Das Aktivitätsdiagram der Loginseite ist in Abbildung~\ref{fig:umlActivityLogin.png} dargestellt.
+Auf der Seite kann man sich entweder direkt mit Username und Passwort anmelden oder auf registrieren klicken.
+Sind die Anmeldedaten korrekt wird man eingeloggt, sind die Anmeldedaten fehlerhaft können diese korrigiert werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityLogin.png}
+    \caption{Aktivitätsdiagramm der Loginseite}
+    \label{fig:umlActivityLogin.png}
+\end{figure}
+
+In Abbildung~\ref{fig:umlActivityRegister.png} wird die Registrierungsseite beschrieben.
+Auf der Seite müssen Daten, wie Vorname, Nachname, E-Mail, Benutzername, sowie Passwort und dessen Wiederholung eingetragen werden.
+Sind die Registrierungsdaten korrekt, bzw. nicht bereits registriert (z.B. Benutzername) kann mit Bestätigung auf registrieren ein User angelegt werden.
+Zu jedem Zeitpunkt kann die Registrierung abgebrochen und zur Loginseite zurückgekehrt werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityRegister.png}
+    \caption{Aktivitätsdiagramm der Registrierungsseite}
+    \label{fig:umlActivityRegister.png}
+\end{figure}
+
+Die Sidebar ist auf jeder Seite mit Außnahme Login-, Registrierungs- und Add-Feedseite erreichbar.
+In Abbildung~\ref{fig:umlActivitySidebar.png} werden die möglichen Aktionen der Sidebar dargestellt.
+Von dieser kann direkt zwischen Dark- und Light-Mode umgeschalten werden. Zudem kann man sich direkt einloggen
+oder falls bereits geschehen auf die Einstellungen gewechselt werden. Unter der Voraussetzung, dass der Benutzer eingeloggt ist kann direkt zu den
+Empfehlungen, den Lesezeichen, den Abonnements und Explore gewechselt werden. Zusätzlich kann direkt ein neuer RSS Feed hinzugefügt werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivitySidebar.png}
+    \caption{Aktivitätsdiagramm der Sidebar}
+    \label{fig:umlActivitySidebar.png}
+\end{figure}
+
+Auf der Übersichtsseite können empfohlene Feeds oder empfohlene Artikel besucht werden.
+Das Aktivitätsdiagramm ist in Abbildung~\ref{fig:umlActivityOverview.png} dargestellt.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityOverview.png}
+    \caption{Aktivitätsdiagramm der Übersichtsseite}
+    \label{fig:umlActivityOverview.png}
+\end{figure}
```

Comment:
Der Pfeil muss von dem viereich in die overview page zeigen
![image](https://user-images.githubusercontent.com/54294591/167505751-fe421463-8da8-43ea-9e54-9210be2bd1d1.png)